### ISSUE TITLE
Showing an image in bdv crashes

### ISSUE THREAD

**Issue by tischi** [fiji/fiji#256]
With the update to 29.0.0 `[BigDataViewer > Open Current Image]` throws this error:
```
[ERROR] Module threw error
java.lang.NoSuchFieldError: list
	at bdv.viewer.BasicViewerState.notifyListeners(BasicViewerState.java:1622)
	at bdv.viewer.BasicViewerState.addSources(BasicViewerState.java:724)
	at bdv.viewer.SynchronizedViewerState.addSources(SynchronizedViewerState.java:496)
	at bdv.viewer.state.ViewerState.<init>(ViewerState.java:81)
	at bdv.viewer.ViewerPanel.<init>(ViewerPanel.java:248)
	at bdv.viewer.ViewerFrame.<init>(ViewerFrame.java:103)
	at bdv.BigDataViewer.<init>(BigDataViewer.java:335)
	at bdv.BigDataViewer.open(BigDataViewer.java:518)
	at bdv.ij.OpenImagePlusPlugIn.run(OpenImagePlusPlugIn.java:133)
	at org.scijava.command.CommandModule.run(CommandModule.java:196)
	at org.scijava.module.ModuleRunner.run(ModuleRunner.java:165)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:124)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:63)
	at org.scijava.thread.DefaultThreadService.lambda$wrap$2(DefaultThreadService.java:225)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```
@tpietzsch

**Comment by tischi** [comment]
Since @tpietzsch is on holidays, maybe @nicokiaru has a clue? He used the new bdv quite a lot already, afiak.

**Comment by tischi** [comment]
I looked a bit into it and I don't get it.

The error occurs in the `BasicViewerState` state class at line 1622 (`bdv-core`):
```
private void notifyListeners( final ViewerStateChange change )
{
	listeners.list.forEach( l -> l.viewerStateChanged( change ) );
}
```
The error says `java.lang.NoSuchFieldError: list`, however debugging within IntelliJ the `list` field does seem to exist:

![image](https://user-images.githubusercontent.com/2157566/89118441-2cd46c80-d4a6-11ea-8d4e-f1842e3c0f20.png)

The only thing I can imagine is some incompatibility with the `listeners`, which are a `Listeners.List` class from `org.scijava.listeners`. I think this used to be a class within `bigdataviewer-core`, so maybe some dependency issue?

@ctrueden, maybe this helps?

**Comment by haesleinhuepf** [comment]
Hey @tischi, can you figure out in which jar file `org.scijava.listeners` lives, which version of it is in your IDE and which version is in your Fiji?

**Comment by tischi** [comment]
Hey @haesleinhuepf,  in both my IDE and Fiji it is: `scijava-listeners-1.0.0-beta-3.jar`.

**Comment by haesleinhuepf** [comment]
Hey @tischi ,

could you try if version 1.0.0-beta4-SNAPSHOT solves the issue?

**Comment by imagejan** [comment]
@tischi I can confirm the issue in an up-to-date Fiji, and with a locally-built `bigdataviewer-core-9.0.6-SNAPSHOT` from latest `master`.

It seems to be a byte-code incompatibility issue, because `bigdataviewer-core` currently depends on `scijava-listeners-1.0.0-beta-2`.

After bumping the version (without any other code changes):

https://github.com/bigdataviewer/bigdataviewer-core/compare/master...imagejan:fix-scijava-listeners

... and dropping the `bigdataviewer-core-9.0.6-SNAPSHOT.jar` built from that into my `Fiji.app/jars/`, the issue appears to be fixed.

I'll prepare a PR to `bigdataviewer-core` bumping the parent version to the latest `pom-scijava`.

**Comment by imagejan** [comment]
As @tpietzsch is on vacation, maybe @ctrueden as second maintainer of `bigdataviewer-core` can review https://github.com/bigdataviewer/bigdataviewer-core/pull/103, cut a new release, and upload to the update site. 🙂

**Comment by ctrueden** [comment]
I released bigdataviewer-core-9.0.6.jar and uploaded to the core update site.

**Comment by apr673** [comment]
> I released bigdataviewer-core-9.0.6.jar and uploaded to the core update site.

Hey, I've just updated ImageJ and I ran into the same error. After checking the version for bigdataviewer it was still the 9.0.5 one. Is it possible that the core update site is not releasing the correct version? (downloading the bigdataviewer-core-9.0.6-SNAPSHOT.jar solved the problem).
Thanks!

**Comment by imagejan** [comment]
@apr673 `bigdataviewer-core-9.0.6.jar` is on the *Java-8* update site. Is it possible that you have other update sites activated that shade this version of `bigdataviewer-core`?
